app/src/androidTest/java/com/github/h3lp3rs/h3lp/MedicalCardAcivityTest.kt

```diff
@@ -0,0 +1,172 @@
+package com.github.h3lp3rs.h3lp
+
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import android.view.View
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.not
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class MedicalCardAcivityTest {
```

Comment:
All the magic numbers and strings in those tests could be put into constants that you declare at the beginning of each test. Even though you use them right below it would make the tests clearer for someone who didn't write the actual code.